sklearn/model_selection/_split.py

```diff
@@ -378,11 +378,11 @@ class KFold(_BaseKFold):
     >>> from sklearn.model_selection import KFold
     >>> X = np.array([[1, 2], [3, 4], [1, 2], [3, 4]])
     >>> y = np.array([1, 2, 3, 4])
-    >>> kf = KFold(n_splits=2)
+    >>> kf = KFold(n_splits=2, random_state=0)
```

Comment:
The `repr` displays a random random_state that fails the test... So I explicitly set one...